
try:
    print(x)
except:
    print("An Exception occurred")    